staff/models/batch.py

```diff
@@ -83,14 +83,14 @@ def ready_for_graduation(self):
         return (self.end_date <= datetime.date.today()) and enrolments_pending_graduation.exists()
 
     def html_formatted_batch_price(self):
-        original_price = settings.CODING_BASICS_REGISTRATION_FEE_SGD
-        html_formatted_price = f"<span class='float-end d-none d-xl-block'>${original_price}</span>"
-        html_formatted_price += f"<div class='lh-lg d-xl-none my-10'>${original_price}<div>"
+        original_price = self.price
+        html_formatted_price = f"<span class='float-end d-none d-xl-block'>S${original_price}</span>"
```

Comment:
Thank you for remembering to use `S$`. We can internationalise this when we need to. Looking forward to that project ;)